src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.nuspec

```diff
@@ -88,9 +89,12 @@
 
     <!-- net9.0 -->
     <file src="net9.0\build\common\MSTest.TestAdapter.props" target="build\net9.0\MSTest.TestAdapter.props" />
-    <file src="net9.0\build\common\MSTest.TestAdapter.targets" target="build\net9.0\MSTest.TestAdapter.targets" />
+    <file src="net9.0\build\uwp\MSTest.TestAdapter.targets" target="build\net9.0\MSTest.TestAdapter.targets" />
```

Comment:
I didn't look yet into the differences between `MSTest.TestAdapter.targets` for regular net9.0 and for net9.0 UWP, but it looks strange to me to pack the UWP one into non-uwp directory

Reply:
Write this should use common not uwp. Bad merge. Reverting.